src/SearchWrapper.cpp

```diff
@@ -0,0 +1,163 @@
+#include <Rcpp.h>
+#include <xapian.h>
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <vector>
+
+using namespace std;
+
+Rcpp::List convertToDataFrame(Rcpp::List list) {
+  Rcpp::List dataFrame = clone(list);
+  Rcpp::GenericVector aRow = dataFrame(0);
+  Rcpp::StringVector rowNames(aRow.length());
+  
+  for (int i = 0; i < aRow.length(); ++i) {
+    char name[5];
+    sprintf(&(name[0]), "%d", i+1);
+    rowNames(i) = name;
+  }
+  
+  dataFrame.attr("row.names") = rowNames;
+  Rcpp::StringVector colNames(dataFrame.length());
+  
+  int ascii=97;
```

Comment:
This is a "magic number", and generally (even when it's fairly obvious what the number means) I'd add a source comment to explain what I'm actually doing here. Something like (if I'm reading this correctly):

```c++
// Name columns from 'a' upwards
```